mne_connectivity/tests/test_envelope.py

```diff
@@ -38,44 +39,53 @@ def _compute_corrs_orig(data):
 def test_envelope_correlation():
     """Test the envelope correlation function."""
     rng = np.random.RandomState(0)
-    data = rng.randn(2, 4, 64)
+    n_epochs, n_signals, n_times = 2, 4, 64
+    data = rng.randn(n_epochs, n_signals, n_times)
+    names = np.arange(n_signals)
     data_hilbert = hilbert(data, axis=-1)
     corr_orig = _compute_corrs_orig(data_hilbert)
     assert (0 <= corr_orig).all()
     assert (corr_orig < 1).all()
+
     # using complex data
-    corr = envelope_correlation(data_hilbert)
-    assert_allclose(corr, corr_orig)
+    corr = envelope_correlation(data_hilbert, names=names)
+    assert_allclose(corr.get_data(), corr_orig.flatten())
+
     # using callable
-    corr = envelope_correlation(data_hilbert,
+    corr = envelope_correlation(data_hilbert, names=names,
```

Comment:
Names should not be obligatory, if not provided it should just use `np.arange(len(signals))`